.github/ISSUE_TEMPLATE.md

```diff
@@ -1,38 +1,17 @@
-### Do you want to request a *feature* or report a *bug*?
--> *Bug*
+## Expected Behavior
 
-----
 
-### What is the current behavior?
--> *Quit accidentally*
+## Actual Behavior
 
----
 
-#### If the current behavior is a bug, please provide the steps to reproduce.
--> *A great way to do this is to provide screenshots and commands.*
+## Steps to Reproduce the Problem
+<!-- Provide screenshots and commands -->
 
---- 
+  1.
+  2.
+  3.
 
-#### What is the expected behavior?
--> 
-
----
-
-### If this is a feature request, what is motivation or use case for changing the behavior?
-
----
-
-### Specs
-#### Wireless adapter:
-*Exact model, chipset, and driver versions.*
-
-
-#### Does it support injection?
-*Paste the output of `aireplay-ng -9`*
-
-
-#### Fluxion version:
-*Paste the output of* `cat fluxion.sh | grep revision`
-
-#### Linux version:
-*Paste the output of* `cat /proc/version` or `uname -r` 
+## Specifications
+<!-- Paste the output of:
+	grep -E 'FLUXIONVersion=|FLUXIONRevision=' fluxion.sh && cat /proc/version && airmon-ng && aireplay-ng wlan0 -9
```

Comment:
@deltaxflux you should add this too because it is one command to get all versions, driver, chipset, etc